packages/vulcan-lib/lib/modules/routes.js

```diff
@@ -61,6 +63,7 @@ export const extendRoute = (routeName, routeProps) => {
   }
 };
 
+if (Picker) Picker.middleware(cookieParser())
```

Comment:
If static routes are getting unparsed cookies, I would suspect a middleware application-order problem. I think static routes and regular requests go through the same middleware stack.

Reply:
I don't think so. The regular requests go through the meteor `server-render` package, which even if it also depends on Picker, would depend on its own instance of Picker.